protocol/lib/eth/util_test.go

```diff
@@ -9,10 +10,104 @@ import (
 	libeth "github.com/dydxprotocol/v4-chain/protocol/lib/eth"
 	"github.com/dydxprotocol/v4-chain/protocol/testutil/constants"
 	bridgetypes "github.com/dydxprotocol/v4-chain/protocol/x/bridge/types"
+	"github.com/ethereum/go-ethereum/accounts/abi"
 	ethcoretypes "github.com/ethereum/go-ethereum/core/types"
 	"github.com/stretchr/testify/require"
 )
 
+func TestGetBridgeEventAbi(t *testing.T) {
+	results := make([]*abi.ABI, 0)
+	mu := &sync.Mutex{}
```

Comment:
nit: consider using the value type.
```suggestion
	mu := sync.Mutex{}
```

the function your using captures `mu` in its closure as a reference to `mu`